client/state/inline-help/actions.js

```diff
@@ -22,6 +23,21 @@ import {
 import getContextualHelpResults from 'state/inline-help/selectors/get-contextual-help-results';
 import 'state/inline-help/init';
 
+/**
+ * Set the query search in the sate tree for the inline help.
+ *
+ * @param {string} searchQuery - query store to store.
```

Comment:
```suggestion
 * @param {string} searchQuery - query string to persist.
```

Use of double "store" was a little confusing.